.pre-commit-hooks.yaml

```diff
@@ -89,3 +89,10 @@
   entry: validate-file-permission-changes
   language: python
   files: ^Packs/
+
+- id: validate-mypy-global-ignore
+  name: validate-mypy-global-ignore
+  description: Validate the changed Python files don't specify global mypy ignore
+  entry: validate-mypy-global-ignore
```

Comment:
`prevent` may be clearer than `validate` imo

Reply:
https://github.com/demisto/demisto-sdk/pull/4405/commits/ba25f585a8a3f6baddb05c1c4abbe1c465d9df2e